### ISSUE TITLE
SharePoint Application

### ISSUE THREAD

**Issue by afabozzi** [rixpoint/testing#1]
I don't know how to send a PM lol.  I am a bit new at the forum stuff and GitHub.  My interest is peaked on the SharePoint app is there a trial for that or is is something for personal use?  thanks.

**Comment by rixpoint** [comment]
I've sent you an email, hopefully you got it mate? I don't think there is a PM function on GitHub to be honest :)

**Comment by afabozzi** [comment]
Yep I got it.  I replied today with a question when you get a chance.  You may check your junk.  Thank you!!
